### PR TITLE
feat(search): Update search dropdown autocomplete design

### PR DESCRIPTION
**Pull request by billyvg**
Removes per-row icons and replaces it with a header instead.

### CODE REVIEW

File: src/sentry/static/sentry/less/shared-components.less [hunk 1]
```diff
@@ -793,84 +793,6 @@ table.table.key-value {
   }
 }
 
-.search-dropdown {
```
Comment (markstory) [inline src/sentry/static/sentry/less/shared-components.less]: :clap:

File: src/sentry/static/sentry/app/components/smartSearchBar/index.jsx [hunk 2]
```diff
@@ -294,7 +294,11 @@ class SmartSearchBar extends React.Component {
         this.setState({loading: false});
         return values.map(value => {
           // Wrap in quotes if there is a space
-          return value.indexOf(' ') > -1 ? `"${value}"` : value;
+          const escapedValue = value.indexOf(' ') > -1 ? `"${value}"` : value;
```
[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/components/smartSearchBar/index.jsx]: https://github.com/getsentry/sentry/pull/12794/commits/19f295c6bd45a4ac51f2e93b7bbc04b423bd1dbe

File: src/sentry/static/sentry/app/components/smartSearchBar/index.jsx [hunk 3]
```diff
@@ -383,18 +404,19 @@ class SmartSearchBar extends React.Component {
...
+        this.updateAutoCompleteState(tagKeys, recentSearches, '', 'tag-key');
         return;
       }
 
       // cursor on whitespace
       // show default "help" search terms
       this.setState({
         searchTerm: '',
-        searchItems: defaultSearchItems,
+        // searchItems: defaultSearchItems,
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/smartSearchBar/index.jsx]: Should this be removed?

File: src/sentry/static/sentry/app/components/smartSearchBar/searchDropdown.jsx [hunk 4]
```diff
@@ -39,39 +48,144 @@ class SearchDropdown extends React.PureComponent {
...
+          <LoadingWrapper key="loading" data-test-id="search-autocomplete-loading">
+            <LoadingIndicator mini={true} />
+          </LoadingWrapper>
+        ) : (
+          <SearchItemsList>
+            {items.map(item => {
+              const isEmpty = item.children && !item.children.length;
+              const invalidTag = item.type === 'invalid-tag';
+
+              // Hide header iff `item.children` is defined and an array and is empty
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/smartSearchBar/searchDropdown.jsx]:
```suggestion
              // Hide header if `item.children` is defined and an array and is empty
```

**Review by markstory — APPROVED** [review]

**Comment by markstory** [comment]
You might need a touch more line-height as descenders are clipped in the suggestions

Screen Shot 2019-04-18 at 9 38 56 AM

**Pull request merged**